<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWithinDialogBase</class>
 <widget class="QDialog" name="SelectWithinDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Within</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select all features from:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsMapLayerComboBox" name="selectFromCombo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Where:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="centroidRadioButton">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Centroid</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Where_Group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pointonsurfaceRadioButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Point on Surface</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Where_Group</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="poleOfInaccessibilityRadioButton">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Pole of Inaccessibility</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Where_Group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="poiTolSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Tolerance. Leave at 1.0 if unsure.</string>
         </property>
         <property name="suffix">
          <string/>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QRadioButton" name="mostlyWithinRadioButton">
         <property name="text">
          <string>Pecrcentage (slow)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Where_Group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="mostlySpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>50.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Is within:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsMapLayerComboBox" name="selectWithCombo"/>
     </item>
     <item>
      <widget class="QCheckBox" name="selectedFeaturesCheckbox">
       <property name="text">
        <string>Using selected features</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="newSelectionRadioButton">
         <property name="text">
          <string>Creating new selection</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Selection_Group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="currentSelectionRadioButton">
         <property name="text">
          <string>Adding to current selection</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">Selection_Group</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="Advanced">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="title">
        <string>Advanced</string>
       </property>
       <widget class="QCheckBox" name="noIndexSelectFrom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>281</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Slower, but might be useful if errors occur.</string>
        </property>
        <property name="text">
         <string>Don't use index on select from layer</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="noIndexSelectWith">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>281</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Slower, but might be useful if errors occur.</string>
        </property>
        <property name="text">
         <string>Don't use index on select with layer</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="noDissolveSelectWith">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>281</width>
          <height>17</height>
         </rect>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Don't dissolve before percentage within selection</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="button_box">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>SelectWithinDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>SelectWithinDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="Where_Group"/>
  <buttongroup name="Selection_Group"/>
 </buttongroups>
</ui>
